src/services/iiif-parser.js

```diff
@@ -168,3 +171,32 @@ export function hasNextSection(index) {
     .map(canvas => canvas.id);
   return canvasIDs.length - 1 > index ? true : false;
 }
+
+/**
+ * Identify the item at the top of the structure
+ * @param {Object} item
+ */
+export function isAtTop(item) {
+  const behavior = getReduxManifest()
+    .getRangeById(item.id)
+    .getBehavior();
+
+  if (behavior && behavior.value === 'top') {
```

Comment:
Looking at the spec again, I don't see this behavior listed in V3.  It is there as a `viewingHint` in V2 though.  So this might work for now, but it seems like something we might want to rethink moving forward.  It could also be that the spec is just missing this behavior by accident.

Reply:
Oh right. It is not there. Yes, maybe it's missing. Or it is renamed in 3.0?